### PR TITLE
[INLONG-10387][Audit]  Audit SDK supports obtaining Audit-Proxy capabilities through InLong Manager

### LINKED ISSUE #10387: [Improve][Audit]  Audit SDK supports obtaining Audit-Proxy capabilities through InLong Manager
### Description

Audit SDK supports obtaining Audit-Proxy capabilities through InLong Manager

### InLong Component

InLong Audit

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by doleyzi**
- Fixes #10387

### Motivation

Audit SDK supports obtaining Audit-Proxy capabilities through InLong Manager

### Modifications

The audit SDK adds a new interface that allows the audit proxy list to be obtained through the manager.

### CODE REVIEW

**Review by justinwwhuang — APPROVED** [review]

File: inlong-audit/audit-sdk/src/main/java/org/apache/inlong/audit/entity/AuditType.java [hunk 1]
```diff
@@ -33,7 +33,7 @@ public enum AuditType {
     KUDU("Kudu"),
     POSTGRES("Postgres"),
     BINLOG("Binlog"),
-    TUBE("Tube"),
+    TUBE_MQ("TUBEMQ"),
```
Comment (aloyszhang) [inline inlong-audit/audit-sdk/src/main/java/org/apache/inlong/audit/entity/AuditType.java:36]:
```suggestion
    TUBE_MQ("TubeMQ"),
```

File: inlong-audit/audit-sdk/src/main/java/org/apache/inlong/audit/send/ProxyManager.java [hunk 2]
```diff
@@ -42,13 +60,77 @@ public static ProxyManager getInstance() {
...
+        LOGGER.info("Audit Proxy API URL: {}", auditProxyApiUrl);
+
+        this.authConfig = authConfig;
+        this.component = component;
+
+        updateAuditProxy();
+
+        if (autoUpdateAuditProxy) {
+            startTimer();
+            LOGGER.info("Auto Update from manager");
```
Comment (aloyszhang) [inline inlong-audit/audit-sdk/src/main/java/org/apache/inlong/audit/send/ProxyManager.java:87]:
```suggestion
            LOGGER.info("Auto update from manager");
```

File: inlong-audit/audit-sdk/src/main/java/org/apache/inlong/audit/send/ProxyManager.java [hunk 3]
```diff
@@ -42,13 +60,77 @@ public static ProxyManager getInstance() {
...
+    }
+
+    public void setAutoUpdateAuditProxy(boolean autoUpdateAuditProxy) {
+        this.autoUpdateAuditProxy = autoUpdateAuditProxy;
+    }
+
+    public void setUpdateInterval(int updateInterval) {
+        this.updateInterval = updateInterval;
+    }
+
```
Comment (aloyszhang) [inline inlong-audit/audit-sdk/src/main/java/org/apache/inlong/audit/send/ProxyManager.java:133]: It seems these methods are not used.

File: inlong-audit/audit-sdk/src/main/java/org/apache/inlong/audit/entity/AuditType.java (hunk 1 above)
Comment (doleyzi) [reply inlong-audit/audit-sdk/src/main/java/org/apache/inlong/audit/entity/AuditType.java]: fixed

File: inlong-audit/audit-sdk/src/main/java/org/apache/inlong/audit/send/ProxyManager.java (hunk 2 above)
Comment (doleyzi) [reply inlong-audit/audit-sdk/src/main/java/org/apache/inlong/audit/send/ProxyManager.java]: Fixed

File: inlong-audit/audit-sdk/src/main/java/org/apache/inlong/audit/send/ProxyManager.java (hunk 3 above)
Comment (doleyzi) [reply inlong-audit/audit-sdk/src/main/java/org/apache/inlong/audit/send/ProxyManager.java:133]: Fixed

**Review by fuweng11 — APPROVED** [review]

**Review by justinwwhuang — APPROVED** [review]

File: inlong-audit/audit-sdk/src/main/java/org/apache/inlong/audit/entity/AuditType.java [hunk 4]
```diff
@@ -32,9 +32,9 @@ public enum AuditType {
     DORIS("Doris"),
     KUDU("Kudu"),
     POSTGRES("Postgres"),
-    BINLOG("Binlog"),
-    TUBE("Tube"),
-    MYSQL("MySQL"),
+    BINLOG("MYSQL_BINLOG"),
+    TUBE_MQ("TubeMQ"),
+    MYSQL("MYSQL_SQL"),
```
Comment (dockerzhang) [inline inlong-audit/audit-sdk/src/main/java/org/apache/inlong/audit/entity/AuditType.java:37]:
```suggestion
    MYSQL("MYSQL_RAW"),
```

Comment (doleyzi) [reply inlong-audit/audit-sdk/src/main/java/org/apache/inlong/audit/entity/AuditType.java:37]: he name MYSQL_SQL is set by the manager. Changing the name will cause the manager to fail to issue it.

**Review by justinwwhuang — DISMISSED** [review]

File: inlong-audit/audit-sdk/README.md [hunk 5]
```diff
@@ -12,15 +12,54 @@ which can ensure that each module is reconciled in accordance with the unified a
 
 ## Usage
 ### Configure Audit Proxy Addresses
-The Audit SDK will summarize the results according to the cycle  
+- The Audit SDK will summarize the results according to the cycle  
 and send them to the ip:port list set by the interface.
+- There are two ways to set the address of the Audit Proxy, namely fixed ip:port and setting the manager address. Please choose one of the methods.
```
Comment (dockerzhang) [inline inlong-audit/audit-sdk/README.md:17]:
```suggestion
- There are two ways to set the address of the Audit Proxy, configure the address directly or get the address from the manager. Please choose one of the methods.

```

File: inlong-audit/audit-sdk/README.md [hunk 6]
```diff
@@ -12,15 +12,54 @@ which can ensure that each module is reconciled in accordance with the unified a
...
+#### Configure Audit Proxy Addresses by fixed ip:port
 If the ip:port of the AuditProxy is fixed, then this interface needs to be called once. 
 If the AuditProxy changes in real time, then the business program needs to call this interface periodically to update
 ```java
-    HashSet<String> ipPortList=new HashSet<>();
+    HashSet<String> ipPortList = new HashSet<>();
     ipPortList.add("0.0.0.0:54041");
     AuditOperator.getInstance().setAuditProxy(ipPortList);
 ```
```
Comment (dockerzhang) [inline inlong-audit/audit-sdk/README.md:25]:
```suggestion
```

```

File: inlong-audit/audit-sdk/README.md [hunk 7]
```diff
@@ -12,15 +12,54 @@ which can ensure that each module is reconciled in accordance with the unified a
...
+#### Configure Audit Proxy Addresses by InLong Manager
+By configuring the InLong Manager's address, module information, and manager certification information, 
+The Audit SDK will automatically fetch the Manager to obtain the address of the Audit Proxy.
+```java
+        String host = "127.0.0.1:8083"; // Host of manager
```
Comment (dockerzhang) [inline inlong-audit/audit-sdk/README.md:30]:
```suggestion
        String host = "127.0.0.1:8083"; // The manager address
```

**Review by aloyszhang — APPROVED** [review]

**Review by dockerzhang — APPROVED** [review]

**Pull request merged**